test/e2e/framework/networking_utils.go

```diff
@@ -199,18 +212,19 @@ func (config *NetworkingTestConfig) EndpointHostnames() sets.String {
 // maxTries == minTries will confirm that we see the expected endpoints and no
 // more for maxTries. Use this if you want to eg: fail a readiness check on a
 // pod and confirm it doesn't show up as an endpoint.
-func (config *NetworkingTestConfig) DialFromContainer(protocol, containerIP, targetIP string, containerHTTPPort, targetPort, maxTries, minTries int, expectedEps sets.String) {
+func (config *NetworkingTestConfig) DialFromContainer(protocol, dialCommand, containerIP, targetIP string, containerHTTPPort, targetPort, maxTries, minTries int, expectedResponses sets.String) {
```

Comment:
You need to update the comment to reflect the new param name, also, what is the content of expectedResponses supposed to remain the same? The new variable name is pretty generic and the old name specifically states that these are endpoints.

Reply:
dialCommand is not documented

Reply:
can we make the dial command more of an enum with documentation?

```
type EchoServerCommand string
var (
  /* some enum for the default behavior?*/
  // EchoHostname tells the echo server to ...
  EchoHostname = "hostname" 
)
```